### DISCUSSION TITLE
GraphHopper offline routing

### DISCUSSION THREAD

**Discussion by francwalter** [devemux86/cruiser#125] [Q&A] [answered]
Hallo
I am new to Cruiser and played a little with it but I don't find any way to download GraphHopper routing files for offline use.
I am a Locus Maps User since long, and there was GraphHopper support earlier, but not anymore. So I asked here about that:
https://forum.locusmap.eu/index.php?topic=8381.0
And somebody told about Cruiser.
But here I don't find it, only in the Routing setting as "GraphHopper offline" but not where to get the offline files.
In discussions I read that there would be download in the Routing menu but not in my Cruiser app (3.0.17 on Android 9).

I found that download page:
https://download.mapsforge.org/graphs
But where to put the zip?

Can somebody tell me or give me a link to a Cruiser help site where I can read it?
Thanks
Frank

**Comment by devemux86** [comment, marked as answer]
Hi @francwalter,

GraphHopper offline graphs can be downloaded with Cruiser's download manager.

You can find it in the menu "**Tools | Download graphs**":

<img src="https://user-images.githubusercontent.com/3484020/230892244-650f91fd-9fcb-4523-bf9b-0a91fb341a8c.jpg" width="50%"/><img src="https://user-images.githubusercontent.com/3484020/230892666-2405ea02-e35f-4e1e-a4bf-653d744bdef9.jpg" width="50%"/>

**Reply by francwalter** [reply]
Thank you very much, I just didn't find it, sorry.

**Comment by 0709wiwiwi** [comment]
**Android Cruiser version and in the Desktop version .**
In the desktop version you can download the GH graph folder at any location.
I do put them in my vector maps download folder. Just remember where to select.

In Android.  See above reply devemux86.
In Android it is even more comfortable. 
This by the map & poi & graphs download tool.

Desktop version operation.
https://youtu.be/y7RxUlVqihk

Android download GH graphs.
https://youtu.be/IZau32-FjtU

**Reply by francwalter** [reply]
Thank, Android version.
